texttospeech/ssml_addresses/README.md

```diff
@@ -0,0 +1,28 @@
+# Google Cloud Text-to-Speech API
+
+This directory contains samples for Google Cloud Text-to-Speech API. The code in this directory aligns with the [Speaking addresses with SSML tutorial](https://cloud-dot-devsite.googleplex.com/text-to-speech/docs/ssml-tutorial).
```

Comment:
I'm sure this is just for staging, but before merging you'll want to change the linked url from cloud-dot-devsite. :)